front/pages/api/v1/w/[wId]/data_sources/[dsId]/tables/index.ts

```diff
@@ -14,30 +14,26 @@ import logger from "@app/logger/logger";
 import { apiError } from "@app/logger/withlogging";
 
 export type ListTablesResponseBody = {
-  tables: {
-    name: string;
-    table_id: string;
-    description: string;
-    schema: CoreAPITableSchema | null;
-  }[];
+  tables: CoreAPITablePublic[];
 };
 
-const UpsertDatabaseTableRequestBodySchema = t.type({
-  table_id: t.union([t.string, t.undefined]),
-  name: t.string,
-  description: t.string,
-  timestamp: t.union([t.number, t.undefined, t.null]),
-  tags: t.union([t.array(t.string), t.undefined, t.null]),
-  parents: t.union([t.array(t.string), t.undefined, t.null]),
-});
+const UpsertDatabaseTableRequestBodySchema = t.intersection([
```

Comment:
cc @albandum looks like this was not taken by npm run docs: I added two optional new params to the route to post a table.

Reply:
You need to update the top jsdoc above the endpoint for the docs to pick it up! I'll push a small commit to your PR to update